other/materials_designer/create_maxwell_disorder.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
<p>Did we not agree to remove Temperature and just call it a "parameter"? Perhaps, proportional to T?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/257/discussion/#comment-2646539685'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>